frontend/src/components/History/index.jsx

```diff
@@ -160,8 +184,30 @@ class History extends React.Component {
         }
         page_sizes.push(history_data.length);
 
+        //Confirmation button
+        let confirm_message = (
+            <Modal
+                centered
+                dialogClassName="history-modal"
+                show={this.state.showConfirm} 
+                onHide={this.toggleConfirm} animation={false}>
+                <Modal.Header closeButton>
+                    <Modal.Title>WARNING</Modal.Title>
```

Comment:
Instead of "WARNING" in all caps, perhaps let's just do "Confirm" or "Confirmation". Thoughts?